pkg/types/openstack/defaults/platform_test.go

```diff
@@ -0,0 +1,136 @@
+package defaults
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/ipnet"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/openstack"
+)
+
+func validPlatform() *openstack.Platform {
+	return &openstack.Platform{
+		Cloud:           "test-cloud",
+		ExternalNetwork: "test-network",
+	}
+}
+
+func validNetworking() *types.Networking {
+	return &types.Networking{
+		NetworkType: "OVNKubernetes",
+		MachineNetwork: []types.MachineNetworkEntry{{
+			CIDR: *ipnet.MustParseCIDR("10.0.0.0/16"),
+		}},
+	}
+}
+
+func TestSetPlatformDefaults(t *testing.T) {
+	cases := []struct {
+		name                string
+		platform            *types.Platform
+		config              *types.InstallConfig
+		networking          *types.Networking
+		expectedLB          configv1.PlatformLoadBalancerType
+		expectedAPIVIPs     []string
+		expectedIngressVIPs []string
+		valid               bool
```

Comment:
I think we can remove this field as it is not used, and, in this context, we are testing whether the defaults are set as expected (if set at all). Whether the platform fields are valid or not is validated in [`installconfig.go`](https://github.com/openshift/installer/blob/main/pkg/types/validation/installconfig.go).

Reply:
Going back at my https://github.com/openshift/installer/pull/9571#pullrequestreview-2703907333, i made it sound like there is some validation there, opps sorry (i.e. I edited the comment with better clarity to avoid confusion 😓).